server/game/cards/03_TWI/units/RoyalGuardAttache.ts

```diff
@@ -0,0 +1,24 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+
+export default class RoyalGuardAttache extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '0598115741',
+            internalName: 'royal-guard-attache'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addTriggeredAbility({
+            title: 'When Played: Deal 2 damage to this unit.',
```

Comment:
For our ability titles we don't include the trigger, cost, or any "if" or "may" phrases since those will all be implied by the prompt itself (e.g., this ability title won't even appear if the right conditions aren't met). So we generally just describe the effect - in this case it would be `title: 'Deal 2 damage to this unit'`